addons/pos_self_order/models/pos_config.py

```diff
@@ -320,6 +320,17 @@ def _get_available_categories(self):
             )
         )
 
+    def get_kitchen_printer(self):
```

Comment:
```suggestion
    def _get_kitchen_printer(self):
```

make it a private method since it doesn't need to be called via rpc.